evap/contributor/templates/contributor_evaluation_form.html

```diff
@@ -64,25 +64,23 @@ <h5 class="card-title">{% trans 'Course data' %}</h5>
                     <h5 class="card-title me-auto">{% trans 'Evaluation data' %}</h5>
                     {% if evaluation.allow_editors_to_edit %}
                         <div>
-                            <button type="button" class="btn btn-sm btn-light mb-3 createAccountRequestModalShowButton">
+                            <button type="button"
+                                    class="btn btn-sm btn-light mb-3 createAccountRequestModalShowButton">
                                 {% trans 'Request creation of new account' %}
                             </button>
                         </div>
                     {% else %}
                         <div>
-                            <button type="button" class="btn btn-sm btn-light changeEvaluationRequestModalShowButton">
-                                {% trans 'Request changes' %}
-                            </button>
+                            <button type="button"
+                                    class="btn btn-sm btn-light changeEvaluationRequestModalShowButton">{% trans 'Request changes' %}</button>
                         </div>
                     {% endif %}
                 </div>
                 {% include 'evaluation_form_general_questionnaires.html' with evaluation_form=form %}
             </div>
         </div>
         <div class="card mb-3">
-            <div class="card-body">
-                {% include 'contribution_formset.html' with formset=formset staff=False editable=editable %}
-            </div>
+            <div class="card-body">{% include 'contribution_formset.html' with formset=formset staff=False editable=editable %}</div>
```

Comment:
I wouldn't want this. You can configure it to add blank lines (not line breaks) before or after certain tags (e.g. `include`), but you cannot configure it to just add line breaks.